splice/queries/adgroup.py

```diff
@@ -0,0 +1,82 @@
+from splice.models import Adgroup
+from sqlalchemy.sql import exists
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.exc import InvalidRequestError
+
+from splice.queries.common import row_to_dict
+
+
+def get_adgroups_by_campaign_id(campaign_id):
+    from splice.environment import Environment
+
+    env = Environment.instance()
+
+    rows = (
+        env.db.session
+        .query(Adgroup)
+        .filter(Adgroup.campaign_id == campaign_id)
+        .order_by(Adgroup.id.asc())
```

Comment:
why asc over desc? does it make sense to have old ones first? I guess it doesn't matter if we are returning all. We can sort differently in UI